benchmarks/maxtext_v5p_model_configs.py

```diff
@@ -0,0 +1,107 @@
+"""
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ """
+"""Shared Benchmark config for v6e orchestrations."""
+
+import os.path
+from benchmarks import xla_flags_library
+from benchmarks.maxtext_trillium_model_configs import MaxTextModel, _add_to_model_dictionary
+
+
+v5p_model_dict = {}
+
+deepseek_v3_fsdp = _add_to_model_dictionary(
+    v5p_model_dict,
+    MaxTextModel(
+        model_name="deepseek_v3_fsdp",
```

Comment:
Please include `v5p` in the `model_name` for benchmarks in this file. The benchmark runner will have issues if there are duplicate model names, including across the v6e, v5e, and v5p files. Something similar is [being done for v5e](https://github.com/AI-Hypercomputer/maxtext/blob/main/benchmarks/maxtext_v5e_model_configs.py#L26), but unfortunately not for v6e

Reply:
good idea, I added for v5p-suffix at model name.